content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -444,9 +444,9 @@ The Automation beta includes the option to create a one-time email campaign, sim
 1. Select the **Export HTML** icon.
    The campaign HTML downloads to the folder specified on your computer
 1. Navigate to the Automation dashboard.
-1. Locate the automation you want to add the exported email to and hover over the action menu.
+1. Locate the Automation you want to add the exported email to and hover over the action menu.
 1. Select **Edit**.
-1. Within the automation, find the email you want to add the content to and click **Edit**.
+1. Within the Automation, find the email you want to add the content to and click **Edit**.
 ![]({{root_url}}/img/edit-email-automation.png "Edit email content")
 8. Select **Blank Template**.
 9. Select the code editor.
```

Comment:
Worth noting that they don't have to use the code editor. If their campaign was created in the Design editor and has Drag and Drop markup, they can also import it into the Design editor, under "Advanced" in the Build tab.

Reply:
Would that be the "Import Drag & Drop HTML" section?

Reply:
Yep!